Examples/arcs-make

```diff
@@ -116,13 +132,14 @@ help:
 	@echo ""
 	@echo "Example: To run tigmint and arcs with myDraft.fa, myReads.fq.gz, and a custom multiplicty range, run:"
 	@echo "	./arcs-make arcs-tigmint draft=myDraft reads=myReads m=[User defined multiplicty range]"
-	@echo "To ensure that the pipeline runs correctly, make sure that the following tools are in your PATH: bwa, tigmint, samtools, arcs (>= v1.0.2), LINKS (>= v1.8.6)"
+	@echo "To ensure that the pipeline runs correctly, make sure that the following tools are in your PATH: bwa, tigmint, samtools, arcs (>= v new version), LINKS (>= v1.8.6)"
 # Version
 version:
-	@echo "arcs-make 1.1"
+	@echo "arcs-make version no"
```

Comment:
```suggestion
```
Can you actually remove the whole version target? I don't think it's necessary (like we decided for ntEdit)